components/eventing-controller/pkg/backend/nats/jetstream/jetstream.go

```diff
@@ -29,12 +29,14 @@ import (
 var _ Backend = &JetStream{}
 
 const (
-	jsHandlerName          = "jetstream-handler"
-	idleHeartBeatDuration  = 1 * time.Minute
-	jsConsumerMaxRedeliver = 100
-	jsConsumerAcKWait      = 30 * time.Second
-	jsMaxStreamNameLength  = 32
-	separator              = "/"
+	jsHandlerName                     = "jetstream-handler"
+	idleHeartBeatDuration             = 1 * time.Minute
+	jsConsumerMaxRedeliver            = 100
+	jsConsumerAcKWait                 = 30 * time.Second
+	jsMaxStreamNameLength             = 32
+	separator                         = "/"
+	NoNatsSubscriptionErr             = "failed to create NATS JetStream subscription for filter: "
+	NoNatsSubscriptionRequeueDuration = 10 * time.Second
```

Comment:
the name is too lengthy and does not convey what it does. Just have it as `requeueDuration`?